config/channels/in-memory-channel/README.md

```diff
@@ -51,3 +51,20 @@ single Dispatcher for all in-memory Channels.
 ```shell
 kubectl get deployment -n knative-eventing imc-dispatcher
 ```
+
+### Namespace Dispatchers
```

Comment:
I wonder if we should not have installation steps into cluster-scoped dispatcher and namespaced dispatcher, so that we could trim down the required permissions if the customer does not want to do namespace dispatchers. Not as part of this PR, but perhaps worthy of an issue to follow up / discuss?

Reply:
yup! Opening an issue...

Reply:
https://github.com/knative/eventing/issues/2637